src/Avalonia.Controls/LayoutTransformControl.cs

```diff
@@ -159,9 +159,16 @@ private void OnUseRenderTransformPropertyChanged(AvaloniaPropertyChangedEventArg
             //       workaround.
 
             var target = e.Sender as LayoutTransformControl;
+            var wasUsedRenderTransform = (bool)e.OldValue;
             var shouldUseRenderTransform = (bool)e.NewValue;
             if (target != null)
             {
+                if (wasUsedRenderTransform)
```

Comment:
should be `hasUsedRenderTransform` perhaps?

Reply:
Yes, you version is better